packages/smooth_app/lib/pages/scan/ml_kit_scan_page.dart

```diff
@@ -84,55 +98,62 @@ class MLKitScannerPageState extends State<MLKitScannerPage> {
       scale = 1 / scale;
     }
 
-    return Scaffold(
-      body: ScannerOverlay(
-        restartCamera: _resumeImageStream,
-        stopCamera: _stopImageStream,
-        model: _model,
-        scannerWidget: Transform.scale(
-          scale: scale,
-          child: Center(
-            child: CameraPreview(
-              _controller!,
-            ),
-          ),
+    return Transform.scale(
+      scale: scale,
+      child: Center(
+        key: ValueKey<bool>(stoppingCamera),
+        child: CameraPreview(
+          _controller!,
         ),
       ),
     );
   }
 
   Future<void> _startLiveFeed() async {
```

Comment:
out of curiosity when is `_startLiveFeed` called?

Reply:
In initState after checking which camera to use.